File: loops/buzz.py

nums = 1
while nums >= 1 and nums <51:
    if nums % 15 == 0:
        print("Fizz\n")
    elif nums % 5 == 0:
        print("Buzz\n")
    elif nums % 3 == 0:
        print("FizzBuzz\n")
    else:
        print(nums)
    nums += 1

File: loops/buzz.c
//santiago L, fizz C

#include <stdio.h> 

int(nums) = 0;
int main(void){
while (nums >0 && nums < 51){
if(nums % 15 ==0){
    printf("FizzBuzz.\n");
}else if(nums % 5 == 0){
    printf("Buzz.\n");
}else if(nums % 3 == 0){
    printf("Fizz.\n");
}else{
    printf("%d\n", nums);
}
nums +=1;
}
    return 0;
}